### PR TITLE
 View data  in table

### PR DESCRIPTION
**Pull request by sachithrakalani**
How to view data in table  :  
Create a JTable
Create a DefaultTableModel
Add Columns to the Model
Add Data to the Model
Set the Model for the JTable

View data in table

### CODE REVIEW

File: src/EmployeeInfo.java [hunk 1]
```diff
@@ -19,6 +21,7 @@ public class EmployeeInfo extends javax.swing.JFrame {
     public EmployeeInfo() {
         initComponents();
         Connect();
+        table_update();
```
Comment (DulanWirajith) [inline src/EmployeeInfo.java]: use camelCase for function naming

File: src/EmployeeInfo.java [hunk 2]
```diff
@@ -30,15 +33,46 @@ public void Connect() {
         try{
             Class.forName("com.mysql.cj.jdbc.Driver");
             con = DriverManager.getConnection("jdbc:mysql://localhost:3306/employee_information_system","root","");
-            System.out.println("Connected");
+            //System.out.println("Connected");
```
Comment (DulanWirajith) [inline src/EmployeeInfo.java]: remove comments

File: src/EmployeeInfo.java [hunk 3]
```diff
@@ -30,15 +33,46 @@ public void Connect() {
...
         } catch (ClassNotFoundException ex) {
             Logger.getLogger(EmployeeInfo.class.getName()).log(Level.SEVERE, null, ex);
         } catch (SQLException ex) {
-            System.out.println("Not Connected ");
+            //System.out.println("Not Connected ");
```
Comment (DulanWirajith) [inline src/EmployeeInfo.java]: remove comments

File: src/EmployeeInfo.java [hunk 4]
```diff
@@ -30,15 +33,46 @@ public void Connect() {
...
             Logger.getLogger(EmployeeInfo.class.getName()).log(Level.SEVERE, null, ex);
         }
     }
     
+    private void table_update() {
```
Comment (DulanWirajith) [inline src/EmployeeInfo.java]: use camelCase for function naming.

**Review by DulanWirajith — CHANGES_REQUESTED** [review]

File: src/EmployeeInfo.java (hunk 2 above)
Comment (sachithrakalani) [reply src/EmployeeInfo.java]: Done

File: src/EmployeeInfo.java (hunk 3 above)
Comment (sachithrakalani) [reply src/EmployeeInfo.java]: Done

File: src/EmployeeInfo.java (hunk 1 above)
Comment (sachithrakalani) [reply src/EmployeeInfo.java]: Done

File: src/EmployeeInfo.java (hunk 4 above)
Comment (sachithrakalani) [reply src/EmployeeInfo.java]: Done

**Review by DulanWirajith — APPROVED** [review]

**Pull request merged**